Seq2SeqNMT/README.md

```diff
@@ -0,0 +1,38 @@
+# Neural Machine Translation on Android
+
+## Introduction
+
+The PyTorch tutorial [NLP FROM SCRATCH: TRANSLATION WITH A SEQUENCE TO SEQUENCE NETWORK AND ATTENTION](https://pytorch.org/tutorials/intermediate/seq2seq_translation_tutorial.html) is an easy-to-follow tutorial that has step-by-step instructions on how to train sequence to sequence encoder and decode networks with an attention mechanism for neural machine translation. Although the quality of the translation between English and French using the model in the tutorial may not be as good as a leading-edge transformer based model, it presents a great example of how to deploy an interesting model trained from scratch in an Android app.
+
+This Android demo app shows:
+
+* The Python code that saves the encoder and attention-enhanced decoder models, trained based on the code in the PyTorch NMT tutorial;
+* The Python code that loads the saved encoder and decoder models, optimizes them and saves again for mobile apps to use;
+* The Android app that uses the encoder and decoder models to do French-English translation.
+
+## Prerequisites
+
+* PyTorch 1.7 or later (Optional)
+* Python 3.8 (Optional)
+* Android Pytorch library 1.6, 1.7 or later
```

Comment:
If you provide links to the pt models that can be download - it is better to specify one version that was used to script them. (As serialization format may change between versions)

Reply:
Great point! Will do.